e2e/remote/remote.go

```diff
@@ -288,13 +308,14 @@ func (c *ctx) remoteList(t *testing.T) {
 	}
 
 	for _, tt := range testPass {
-		argv := []string{"remote", "--config", config.Name(), "list"}
-		t.Run(tt.name, func(t *testing.T) {
-			fmt.Println("config.name is ", config.Name())
```

Comment:
Note: I did not port this print statement into a log statement because this looks like something that was added for debugging and unnoticed during review.